### PR TITLE
Fix 0.6 warnings and bump up REQUIRE to 0.5

### PR DESCRIPTION
**Pull request by cstjean**

### CODE REVIEW

File: src/pvtree.jl [hunk 1]
```diff
@@ -38,9 +38,9 @@ function buildgraph!(parent::Node, bt::Vector{Vector{UInt}}, counts::Vector{Int}
         end
     end
     ngroups = length(dorder)
-    group = Array(Vector{Int}, ngroups)  # indices in bt that have the same sortorder
-    n = Array(Int, ngroups)              # aggregated counts for this group
-    order = Array(Int, ngroups)
+    group = Array{Vector{Int}}(ngroups)  # indices in bt that have the same sortorder
```
Comment (timholy) [inline src/pvtree.jl]: Even better as `Vector{Vector{Int}}`

File: REQUIRE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-julia 0.4
+julia 0.5
```
Comment (timholy) [inline REQUIRE]: .travis.yml also needs updating

**Comment by cstjean** [comment]
Fixed.

**Comment by timholy** [comment]
Thanks!

**Pull request merged**